tests/scattering1d/test_filters_scattering1d.py

```diff
@@ -229,3 +229,34 @@ def test_compute_temporal_support():
     with pytest.warns(UserWarning) as record:
         compute_temporal_support(h_f)
     assert "too small to avoid border effects" in record[0].message.args[0]
+
+
+def test_filter_bank_length():
+    N = 2**13
+    J = 5
+    T = 2**J
+    Q = 8
+    _, psi1_f, _, _ = scattering_filter_factory(np.log2(N), J, Q, T, spinned=False)
+    _, psi1_f_spinned, _, _ = scattering_filter_factory(np.log2(N), J, Q, T, spinned=True)
+    assert len(psi1_f)*2 == len(psi1_f_spinned)
+
+
+def test_filter_bank_sum():
+    N = 2**13
+    J = 5
+    T = 2**J
+    Q = 8
+    
+    _, psi1_f, _, _ = scattering_filter_factory(np.log2(N), J, Q, T, spinned=False)
+    _, psi1_f_spinned, _, _ = scattering_filter_factory(np.log2(N), J, Q, T, spinned=True)
+    probe_frequency = int((7/8)*psi1_f[0][0].shape[0])
+    psi1_f = np.asarray([psi_f[0] for psi_f in psi1_f])
+    psi1_spinned_f = np.asarray([psi_f[0] for psi_f in psi1_f_spinned])
+    
```

Comment:
```
assert psi1_spinned_f.shape[0] == 2*psi1_f.shape[0]
```

also it would be good to test the `xi` metadata parameter. (sign of xi is sign of spin)

Reply:
test that first 50% of psi_f list have xi>0 and the second 50% have xi<0

Reply:
gotcha